managed/ui/src/components/universes/UniverseForm/ClusterFields.js

```diff
@@ -1629,7 +1639,9 @@ export default class ClusterFields extends Component {
       );
     }
 
-    universeProviderList.unshift(<option key="" value=""></option>);
+    if (universeProviderList.length == 0) {
+      universeProviderList.unshift(<option key="" value=""></option>);
+    }
```

Comment:
I think we should remove this entirely. Don't insert an empty option under any circumstances. If there is no provider, the form should fail.